binaries/speed_benchmark_torch.cc

```diff
@@ -37,12 +37,13 @@ C10_DEFINE_string(
     "semicolon to separate the dimension of different "
     "tensors.");
 C10_DEFINE_string(input_type, "", "Input type (uint8_t/float)");
+C10_DEFINE_string(input_format, "", "Input memory format (NHWC/NCHW)");
```

Comment:
This parameter not having a default value means AIBench invocations must change or they will error.  I think it's better to make this explicit (hence why I opted for not having a default value.)

Reply:
There are too many places this will require update, especially in AIBench. Plus it also introduce discrepancy between this binary and our internal binary that depends on lite predictor. I strongly suggest we keep the default value of NCHW. Otherwise all full JIT runs will be broken.

Reply:
Thanks, will default to NHWC.

Reply:
Note that, that format is valid only for tensors with dims=4. So it wont work for all cases we have in benchmarks. Probably best to do NCHW.

Reply:
I agree to default to ContiguousFormat.  Also, I think the names should be changed to contiguous_format and channels_last, since that's the terminology that PyTorch uses.

Reply:
Done